lab-teddy/tests/client-test.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const expect = require('chai').Expect;
+const testClient = require('../model/client.js');
+
+describe('client test', () => {
+  before('Create Client object', () =>{
+    this.client = new testClient();
```

Comment:
close...but this is not how you instantiate a client. Doesn't your constructor take a socket? How would you mock that socket? After you do that, you should be able to test the properties in the it blocks! Keep going! you're almost there man!!